### PR TITLE
Aarch64 enablement

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by ancorgs** [comment]
Why reformatting the .spec so intensively? I would prefer if only the real meaningful changes would be included.

File: src/clients/virtualization.rb [hunk 1]
```diff
@@ -76,13 +76,17 @@ def main
...
+      is_aarch64 = false
 
       # check whether VM can be started (cannot start a vm using UML)
       return false if VirtConfig.isUML
 
       # s390 is technical preview and we only fully support x86_64
       if Arch.s390_64 == true
         is_s390 = true
+      # aarch64 is technical preview and we only fully support x86_64
+      elsif Arch.aarch64 == true
```
Comment (ancorgs) [inline src/clients/virtualization.rb]: This change looks quite pointless at first sight. It seems to set a local variable that is not used for any purpose. Am I overlooking something?

**Comment by ihno** [comment]
Dirk is just using the same way as it was done for s390x.
You have a variable you have to find out whether you are on the specific architecture.
However the variable should be of course global.
Can you please fix that?

File: src/clients/virtualization.rb (hunk 1 above)
Comment (dirkmueller) [inline src/clients/virtualization.rb]: The point is to avoid the return false on line 91

**Comment by dirkmueller** [comment]
@ancorgs thats why I made two commits, please take a look at the 2nd commit for the relevant change:
https://github.com/yast/yast-vm/pull/12/commits/8c1a73a4cd2d035ff47d79545eea2f71130a6e54

File: src/clients/virtualization.rb (hunk 1 above)
Comment (ancorgs) [inline src/clients/virtualization.rb]: I didn't mean the condition of the `if`, but the line below `is_aarch64 = true` because `is_aarch64` is not used.

**Comment by ancorgs** [comment]
We don't run spec-cleaner directly in the repo because it's already part of our auto-submission mechanism. I will prepare an additional version of this pull request right away. Stay tuned.

**Comment by ancorgs** [comment]
Created #13 which should do the same and (at least for my taste) clean-ups the code a little bit.

**Comment by dirkmueller** [comment]
thx

**Pull request closed without merging**